### ISSUE TITLE
Cavitating leukodystrophy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6492]
**Preferred term label:**
Cavitating leukodystrophy

**Synonyms**

**Definition (free text, please give PubMed ID)**
Find a precise definition

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619061 
https://pubmed.ncbi.nlm.nih.gov/25175347/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Actually we already have a term HP_0033369
